tests/Unit/Evolution/Systems/Cce/AnalyticSolutions/Test_GaugeWave.cpp

```diff
@@ -0,0 +1,38 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "Framework/TestingFramework.hpp"
+
+#include <complex>
+#include <cstddef>
+
+#include "Evolution/Systems/Cce/AnalyticSolutions/GaugeWave.hpp"
+#include "Framework/Pypp.hpp"
+#include "Framework/SetupLocalPythonEnvironment.hpp"
+#include "Framework/TestHelpers.hpp"
+#include "Helpers/DataStructures/MakeWithRandomValues.hpp"
+#include "Helpers/Evolution/Systems/Cce/AnalyticSolutions/AnalyticDataHelpers.hpp"
+#include "PointwiseFunctions/GeneralRelativity/Tags.hpp"
+
+namespace Cce::Solutions {
+SPECTRE_TEST_CASE("Unit.Evolution.Systems.Cce.GaugeWave", "[Unit][Cce]") {
+  MAKE_GENERATOR(gen);
+  UniformCustomDistribution<double> radius_dist{5.0, 10.0};
+  UniformCustomDistribution<double> parameter_dist{0.5, 2.0};
+  const double extraction_radius = radius_dist(gen);
+  const size_t l_max = 16;
+  // use a low frequency so that the dimensionless parameter in the metric is ~1
```

Comment:
just because it adds another lengthscale into the problem. if you have a high frequency, you have terms of different scales and make it harder to get a tight approximation in the test.

Reply:
Ah okay,thank you!